docs/using-mycroft-ai/basic-commands.md

```diff
@@ -1,104 +1,328 @@
+> Hey Mycroft, record me for 6 seconds
+##### Playback 
+> Hey Mycroft, play record <br>
+> Hey Mycroft, play back the audio <br>
+> Hey Mycroft, playback recording
+##### Delete the file
+> Hey Mycroft, delete audio <br>
+> Hey Mycroft, erase recording <br>
+> Hey Mycroft, record clear
+###
+### Configuration - *Interacts with the Device configuration both locally and at home.mycroft.ai.*
+##### Query the registered **name** of this Mycroft Device
+> Hey Mycroft, what is your name?
+##### Query the registered location of this Mycroft Device
+> Hey Mycroft, where are you? <br>
+> Hey Mycroft, tell me your state
+##### Query which listener is being used to detect the Wake Word
+> Hey Mycroft, tell me the wake word listener <br>
+> Hey Mycroft, what is the listener?
+##### Command Mycroft to update the configuation both locally and at home.mycroft.ai	
+> Hey Mycroft, configuration update<br>
+> Hey Mycroft, update config
+##### For developers: Select which listener to use
+> Hey Mycroft, set the listener to precise<br>
+> Hey Mycroft, set the listener to default
+##### For developers: Enable or disable the development version of Precise listener
+> Hey Mycroft, enable the precise dev model <br>
+> Hey Mycroft, disable the new precise model
+###
+### Hello World - *Pleasantries with Mycroft.*
+> Hey Mycroft, greetings <br>
+> Hey Mycroft, hello world
+> 
+> Hey Mycroft, thank you
+> 
+> Hey Mycroft, how are you doing? <br>
+> Hey Mycroft, how was your day?
+
+*Note: Hello World is a well-documented skill that serves as a starting point for skill code development.*
+###
+### IP - *Query the SSID or the IP Address (aka the network address) of the Mycroft Device.*
+##### Query Mycroft's IP address
+> Hey Mycroft, what is your IP address? <br>
+> Hey Mycroft, tell me your network address <br>
+> Hey Mycroft, what network are you connected to?
+##### Query the SSID
+> Hey Mycroft, what SSID are you on? <br>
+> Hey Mycroft, what wifi network are you using? <br>
+> Hey Mycroft, what network are you connected to?
+###
+### Jokes - *Mycroft uses pyjokes as the source of jokes to tell.*
+> Hey Mycroft, tell me a joke <br>
+> Hey Mycroft, make me laugh <br>	
+> Hey Mycroft, brighten my day with a Chuck Norris joke <br>
+> Hey Mycroft, tell me a non-offensive joke <br>
+###
+### Naptime - *While 'napping' Mycroft will ignore utterances other than "Wake up".*
+##### Begin nap
+> Hey Mycroft, go to sleep <br>
+> Hey Mycroft, nap time
+##### End nap
+> Hey Mycroft, wake up
+###
+### NPR News - *Plays the RSS audio feed for the requested station, defaulting to NPR if none specified or it is unknown.*
+##### Play the news
+> Hey Mycroft, play the BBC news <br>
+> Hey Mycroft, what are the headlines?
+##### Restart the broadcast
+> Hey Mycroft, start over <br>
+> Hey Mycroft, restart news
+##### Stop playing the news
+> Hey Mycroft, stop
+
+*Known stations include: ABC (Australia), AP, BBC, CBC, DLF, Ekot, FOX, GPB, NPR, PBS, RDP, RNE, TSF, OE3, VRT, WDR, YLE.*
```

Comment:
again, it feels like this should be up under the heading as part of the description

Reply:
Right.